pkg/scanners/clairify/convert_test.go

```diff
@@ -72,7 +72,7 @@ func TestConvertNodeToVulnRequest(t *testing.T) {
 			KubeletVersion:   testCase.kubeletVersion,
 			KubeProxyVersion: testCase.kubeProxyVersion,
 		}
-		assert.Equal(t, testCase.expected, convertNodeToVulnRequest(node))
+		assert.Equal(t, testCase.expected, convertNodeToVulnRequest(node, nil))
 	}
 }
```

Comment:
It might be good to add a test case for conversion with node inventory too.

Reply:
Thanks! Got that covered in [ae4a5ff](https://github.com/stackrox/stackrox/pull/4492/commits/ae4a5ff533a6db77a95c066d891881a8ca31a0c3)